src/Behat/Testwork/ServiceContainer/ExtensionManager.php

```diff
@@ -131,11 +131,9 @@ public function debugInformation()
     /**
      * Attempts to guess full extension class from relative.
      *
-     * @param string $locator
-     *
-     * @return string
+     * @internal
      */
-    private function getFullExtensionClass($locator)
+    public static function guessFullExtensionClassName(string $locator): string
```

Comment:
This is a pure function, it needs to be public for the "convert shorthand to FQCN extension class" feature. I've kept that as a separate commit in case it needs further discussion.